projectmanagement/src/main/java/life/qbic/projectmanagement/domain/project/ProjectIntent.java

```diff
@@ -39,12 +47,11 @@ public boolean equals(Object o) {
 
     ProjectIntent that = (ProjectIntent) o;
 
-    return projectTitle.equals(that.projectTitle);
+    return projectTitle.equals(that.projectTitle) && projectObjective.equals(that.projectObjective);
```

Comment:
How do we need to calculate this mehtod according to ddd? @KochTobi

Reply:
yes you can remove the equals and hash code methods and regenerate them.
Be sure that all fields are included and marked as non-null